docs/cookbook/listening_to_core_events.md

```diff
@@ -118,16 +118,16 @@ class SignalListener implements EventSubscriberInterface
      */
     private $logger;
 
-    public function __construct( LoggerInterface $logger )
+    public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
     }
 
-    public function onAPISignal( SignalEvent $event )
+    public function onAPISignal(SignalEvent $event)
     {
         $signal = $event->getSignal();
         // You may want to check the signal type here to react accordingly
-        $this->logger->debug( 'Received Signal: ' . print_r( $signal, true ) );
+        $this->logger->debug( 'Received Signal: ' . print_r($signal, true) );
```

Comment:
Nitpick: `$this->logger->debug( 'Received Signal: ' . print_r($signal, true) );` -> `$this->logger->debug('Received Signal: ' . print_r($signal, true));`